src/test/java/tools/jackson/databind/ext/javatime/ser/WriteDatesAsTimestamps5142JavaTimeTests.java

```diff
@@ -0,0 +1,160 @@
+package tools.jackson.databind.ext.javatime.ser;
+
+import java.time.*;
+
+import org.junit.jupiter.api.Test;
+import tools.jackson.databind.ObjectMapper;
+import tools.jackson.databind.cfg.DateTimeFeature;
+import tools.jackson.databind.ext.javatime.DateTimeTestBase;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+// [databind#5142] JSTEP-5 Tests to verify current behavior and discuss further action DateTimeFeature.WRITE_DATES_AS_TIMESTAMPS
+public class WriteDatesAsTimestamps5142JavaTimeTests
+    extends DateTimeTestBase
+{
+    private static final ObjectMapper WITH_TIMESTAMP_MAPPER = mapperBuilder()
+            .enable(DateTimeFeature.WRITE_DATES_AS_TIMESTAMPS)
+            .build();
+
+    private static final ObjectMapper WITHOUT_TIMESTAMP_MAPPER = mapperBuilder()
+            .disable(DateTimeFeature.WRITE_DATES_AS_TIMESTAMPS)
+            .build();
+
+    @Test
+    public void testWriteDatesAsTimeStamps() throws Exception {
+        // java.time.OffsetDateTime
+        _testTimestamp(
+                LocalDateTime.of(2025, 5, 4, 18, 1, 0),
+                LocalDateTime.class,
+                // Expected "[2025,5,4,18,1,0]",
+                "[2025,5,4,18,1]", // Acutal
```

Comment:
Problem with "local" date/time values is that there is no Timestamp: they cannot be represented same as Offset/Zoned date/time values (or java.util.Calendar).

So I think "timestamp" for config is more like "numeric" vs "textual".

Reply:
It looks we are missing last 1 or 2 values (seconds, and even nanoseconds).
But maybe it is because last number is `0` and it's omitted?